tests/ui/check-cfg/well-known-values.rs

```diff
@@ -1,41 +1,90 @@
-// This test check that we lint on non well known values and that we don't lint on well known
-// values
+// This test check that we recogniez all the well known config names
+// and that we correctly lint on unexpected values.
+//
+// This test also serve as an "anti-regression" for the well known
+// values since the suggestion shows them.
 //
 // check-pass
 // compile-flags: --check-cfg=cfg() -Z unstable-options
 
+#![feature(cfg_overflow_checks)]
+#![feature(cfg_target_has_atomic)]
+#![feature(cfg_target_has_atomic_equal_alignment)]
+#![feature(cfg_target_abi)]
+#![feature(cfg_relocation_model)]
+#![feature(cfg_sanitize)]
+#![feature(cfg_target_thread_local)]
+
+#[cfg(any(
+    doc = "_UNEXPECTED_VALUE",
+    //~^ WARN unexpected `cfg` condition value
```

Comment:
I would put this list in alphabetical order, and maybe even add `tidy-alphabetical-{start,end}` markers so it is guaranteed to stay in order.

Reply:
Great, and to make sure it works as expected I've tested it locally with success.